contracts/ICO_crowdsale.sol

```diff
@@ -111,8 +111,11 @@ contract WhitelistedCrowdsale is Ownable, ERC223Receiver {
         if (weiAmount < 10 ether) {
 
         } else if (weiAmount < 25 ether) {
-            bonusAmount = basicAmount.div(10);
+            if (!isPrivateOffer){
```

Comment:
Is this intentional that we don't add 10% for private offer? I don't see this logic in the whitepaper.